AbbreviationAsWordInName/Example2/list-of-projects.properties

```diff
@@ -38,7 +38,9 @@ jOOL|git|https://github.com/jOOQ/jOOL|version-0.9.7||
 RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
 Vavr|git|https://github.com/vavr-io/vavr|v0.9.0||
 
-# Few projects with excludes to decrease a number of checked files (usefull for some checks which specify overly strong code style policy)
+# For special checks only
+
+# Few projects with excludes to decrease a number of checked files (useful for some checks which specify overly strong code style policy)
```

Comment:
remove all lines below this line.
all of them are "duplicates" of lines above, and we kep them just for convenence to enable when required.

Reply:
can you once review again?

Reply:
as soon as you write "done", I will review for sure.